### PR TITLE
Internal: Added GoPro link URL Kit Library UTM params [ED-8547]

### PR DESCRIPTION
**Pull request by davseve**
## PR Checklist

- [ ] The commit message follows our guidelines:  https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ x ] Other... Please describe: Internal

## Summary

This PR can be summarized in the following changelog entry:

*

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

### CODE REVIEW

**Comment by arielk** [comment]
@davseve v3.8 is already released, please send this PR to the develop branch

**Comment by louiswol94** [comment]
@davseve Can you add a test for this?

File: app/modules/kit-library/assets/js/hooks/use-kit-call-to-action.js [hunk 1]
```diff
@@ -5,12 +5,17 @@ export const TYPE_CONNECT = 'connect';
...
 	const { settings } = useSettingsContext();
 
 	// SubscriptionPlan can be null when the context is not filled (can be happened when using back button in the browser.)
 	const subscriptionPlan = useMemo( () => settings.subscription_plans?.[ kitAccessLevel ], [ settings, kitAccessLevel ] );
 
+	// const updatePromotionUrl = () => {
+	// 	const cleanTitle = kitTitle.replace( /\s/g, '-' ).replace( /[^\w-]/g, '' ).toLowerCase();
+	// 	return subscriptionPlan.promotion_url += `&utm_term=${ cleanTitle }&utm_content=${ kitId }`;
+	// };
+
```
Comment (louiswol94) [inline app/modules/kit-library/assets/js/hooks/use-kit-call-to-action.js:18]: @davseve Can you delete this commented out code? it is causing a JS lint error.

File: app/modules/kit-library/assets/js/hooks/use-kit-call-to-action.js [hunk 2]
```diff
@@ -5,12 +5,17 @@ export const TYPE_CONNECT = 'connect';
 export const TYPE_PROMOTION = 'promotion';
 export const TYPE_APPLY = 'apply';
 
-export default function useKitCallToAction( kitAccessLevel ) {
+export default function useKitCallToAction( kitAccessLevel, kitTitle, kitId ) {
```
Comment (louiswol94) [inline app/modules/kit-library/assets/js/hooks/use-kit-call-to-action.js:8]: @davseve Maybe just double check these params as they are never used?

**Review by louiswol94 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**